project-exe/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(${PROJECT_NAME} main.cpp)
+add_executable(${PROJECT_NAME} main.cpp ../include/benchmark/IBenchmark.h)
```

Comment:
sa nu folosesti path relativ. ai ${PROJECT_SOURCE_DIR} care iti e echivalent cu baza proiectului. si nu aici trebuie link-uit header-ul. o sa adaugam alta librarie de benchmarking in folder-ul din /src/benchmark

Reply:
got it🗿